src/Api/requestInboxAlert.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
```

Comment:
This is the our old version of the API. Use the Restfull one. its called `Kirby Refactor`  in postman collections